architecture/architecture-component-imagestreams.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
```

Comment:
It looks like we can update the existing module(s) and include them here:

include::modules/images-imagestream-use-image-streams-managing.adoc[leveloffset=+1]

(There are more modules for this section. I'll add them later.)